src/App.jsx

```diff
@@ -14,70 +16,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  function setHeading(selectedGood) {
+    return goods.includes(selectedGood)
+      ? `${selectedGood} is selected`
+      : 'No goods selected';
+  }
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const [selectedGood, setselectedGood] = useState('Jam');
+  let key = 0;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? (
+          setHeading(selectedGood)
+        ) : (
+          setHeading(selectedGood)
+        )}
```

Comment:
So you are doing exactly the same if selectedGood is truthy and falsy. Just call `setHeading(selectedGood)` once